src/components/ItemFilters/ItemFilters.test.tsx

```diff
@@ -0,0 +1,145 @@
+import React from "react"
+import { render, screen, act } from "@testing-library/react"
+import mockRouter from "next-router-mock"
+
+import FiltersContainer from "./FiltersContainer"
+import userEvent from "@testing-library/user-event"
+
+// Mock next router
+jest.mock("next/router", () => jest.requireActual("next-router-mock"))
+
+describe("Filters container", () => {
+  it("renders three filter boxes", () => {
```

Comment:
Can you add one with only one "agg" to show it can render dynamically not just three but also one (or two) depending on the API response.